src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -36,6 +47,7 @@ PageWrapper.propTypes = {
   isAmp: bool.isRequired,
   pageType: string.isRequired,
   service: string.isRequired,
+  dials: shape.isRequired,
```

Comment:
Shape here is a function
```suggestion
  dials: shape({}).isRequired,
```

Reply:
Chatted with James IRL, could we maybe use `objectOf(bool)`?

Reply:
As mentioned above I have changed this to `objectOf(string)`